third-part-samples/shardingsphere-sample/shardingsphere-jdbc-5.x-spring-sample/src/main/resources/application.yml

```diff
@@ -1,9 +1,5 @@
 spring:
   shardingsphere:
-    database:
-      name: logic_db
-    mode:
-      type: Memory
```

Comment:
- [ ] Remove this to guarantee compatibility with ShardingSphere JDBC 5.1.3 in advance.